lib/KDropdownMenu.vue

```diff
@@ -34,6 +35,11 @@
       UiMenu,
     },
     props: {
+       // Override default prop value
+      constrainToScrollParent: {
+        type: Boolean,
+        default: false
```

Comment:
For consistency, I think this needs to default to `true` as is with Keen UI. We can then override the default if we need to by passing `false`. This will also ensure that we are backward compatible too.

Reply:
There are also, 1 or 2 linting issues to take care of. You can fix any linting issues by running `yarn run lint-fix` before adding and committing your changes

Reply:
For changelog build failures, please copy the changelog below(I made a few changes to your changelog from the pr) and add it to the `CHANGELOG.md` just below the `## Version 3.x.x (release-v3 branch)` section as the latest entry, since your fix is targeting `release-v3`

```markdown
- [#586]
  - **Description:** Adds a new prop `constrainToScrollParent ` to `KDropdownMenu` to allow overriding of its popover flipping behavior.
  - **Products impact:** Bugfix
  - **Addresses:** [#432](https://github.com/learningequality/kolibri-design-system/issues/432)
  - **Components:** KDropdownMenu
  - **Breaking:** no
  - **Impacts a11y:**  no
  - **Guidance:** Use the `constrainToScrollParent` prop to override the default popover flipping behavior of the `KDropdownMenu` component prop where necessary.

[#586]: https://github.com/learningequality/kolibri-design-system/pull/586
```

Reply:
Please be sure to let me know in case you need any support in the process. Thanks again